x-pack/solutions/security/plugins/elastic_assistant/server/routes/evaluate/post_evaluate.ts

```diff
@@ -159,6 +164,14 @@ export const postEvaluateRoute = (
           // Get a scoped esClient for esStore + writing results to the output index
           const esClient = ctx.core.elasticsearch.client.asCurrentUser;
 
+          // Create output index for writing results and write current eval as RUNNING
+          await setupEvaluationIndex({ esClient, logger });
```

Comment:
Not relevant to this PR, just for my overall understanding :-)

I see that we can request to evaluate multiple graphs which will be translated into the graph names (line 202 in this file). Then I see that we check whether `defendInsightsGraphs` or `attackDiscoveryGraphs` specified and return if one of them is not empty.

Let's say I passed `graphs: ['assistant', 'attack-discovery', 'defend-insights']` in request body. Does that mean we will not evaluate `'assistant'` and `'attack-discovery'` because `'defend-insights'` specified as well? Just trying to understand the logic in this route handler.

Reply:
Yeah, this is something I noticed as well while working this... 😅  I hadn't been in `post_evaluate` in a bit, but seems this change happened sometime when the other graphs got introduced?

I think it would be nice to support running evals for all provided graphs as the API spec allows, but don't necessarily want to make that logic change now, so maybe I just update the spec/UI to only allow sending a single graph for the time being? There's plenty more to revisit around evals and writing results still, so this can be done as needed.

Reply:
I've fixed for now by setting a min/max of `1` for `graphs` on the `post_evaluate` OAS, and also updated the client UI graph selection combobox to be `singleSelection`. This should be good for now 👍 


[2f0284d](https://github.com/elastic/kibana/pull/222487/commits/2f0284d3f94e2e0c87383debbf2d24b367f9a9de)